src/main/java/org/wso2/lsp4intellij/utils/FileUtils.java

```diff
@@ -249,12 +253,7 @@ public static String sanitizeURI(String uri) {
      * @return The virtual file
      */
     public static VirtualFile URIToVFS(String uri) {
-        try {
-            return LocalFileSystem.getInstance().findFileByIoFile(new File(new URI(sanitizeURI(uri))));
-        } catch (URISyntaxException e) {
-            LOG.warn(e);
-            return null;
-        }
+        return virtualFileFromURI(uri);
```

Comment:
This method appears to be a duplicate of `virtualFileFromURI()` unless I am missing something.